contracts/utils/SafeCast.sol

```diff
@@ -94,4 +94,17 @@ library SafeCast {
         require(value < 2**8, "SafeCast: value doesn\'t fit in 8 bits");
         return uint8(value);
     }
+
+    /**
+     * @dev Returns the unsigned uint256 from int256, reverting if
+     * the input is negative.
+     *
+     * Requirements:
+     *
+     * - input must be greater or equal than 0.
```

Comment:
```suggestion
     * @dev Converts an unsigned uint256 into a signed int256.
     *
     * Requirements:
     *
     * - input must be greater than or equal to 0.
```

I think this brings this a bit more in line with the others

Reply:
Didn't notice when I accepted this suggestion that it's actually reversed: it converts a signed int256 into an unsigned uint256.
I just realized when I was implemented the uint->int.